### DISCUSSION TITLE
Dịch generator/discriminator

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#113] [Terminology] [answered]
Hai từ này ở d2l được dịch là `bộ sinh` và `bộ phân biệt`.

cc @mlbvn/handson-ml

**Comment by thanhcsf** [comment]
Vậy em đang concern gì nhỉ?

**Reply by minhduc0711** [reply]
ah em không có vấn đề gì, em chỉ đưa vào đây trước khi đưa vào glossary thôi. Chắc lần sau đưa thẳng vào glossary luôn :v

**Reply by thanhcsf** [reply]
Ừa, đưa vào đi em.

**Comment by minhduc0711** [comment, marked as answer]
Chốt như https://github.com/mlbvn/handson-ml2-vn/discussions/113#discussion-3532594
